actions.go

```diff
@@ -0,0 +1,247 @@
+	Description string          `graphql:"description"`
+	Filter      FilterId        `graphql:"filter"`
+	Id          graphql.ID      `graphql:"id"`
+	Name        string          `graphql:"name"`
+	Owner       TeamId          `graphql:"owner"`
+	Timestamps  Timestamps      `graphql:"timestamps"`
+}
+
+type CustomActionsExternalActionsConnection struct {
+	Nodes      []CustomActionsExternalAction
+	PageInfo   PageInfo
+	TotalCount int
+}
+
+type CustomActionsTriggerDefinitionsConnection struct {
+	Nodes      []CustomActionsTriggerDefinition
+	PageInfo   PageInfo
+	TotalCount int
+}
+
+type CustomActionsWebhookActionCreateInput struct {
+	Name              string                      `json:"name"`
+	Description       *graphql.String             `json:"description,omitempty"`
+	LiquidTemplate    string                      `json:"liquidTemplate"`
+	WebhookURL        string                      `json:"webhookUrl"`
+	HTTPMethod        CustomActionsHttpMethodEnum `json:"httpMethod"`
+	BasicAuthUserName *graphql.String             `json:"basicAuthUserName,omitempty"`
+	BasicAuthPassword *graphql.String             `json:"basicAuthPassword,omitempty"`
+	Headers           JSON                        `json:"headers,omitempty"`
+}
+
+type CustomActionsWebhookActionUpdateInput struct {
+	Id                graphql.ID                  `json:"id"`
+	Name              *graphql.String             `json:"name,omitempty"`
+	Description       *graphql.String             `json:"description,omitempty"`
+	LiquidTemplate    *graphql.String             `json:"liquidTemplate,omitempty"`
+	WebhookURL        *graphql.String             `json:"webhookUrl,omitempty"`
+	HTTPMethod        CustomActionsHttpMethodEnum `json:"httpMethod,omitempty"`
+	BasicAuthUserName *graphql.String             `json:"basicAuthUserName,omitempty"`
+	BasicAuthPassword *graphql.String             `json:"basicAuthPassword,omitempty"`
+	Headers           JSON                        `json:"headers,omitempty"`
+}
+
+type CustomActionsTriggerDefinitionCreateInput struct {
+	Name        string          `json:"name"`
+	Description *graphql.String `json:"description,omitempty"`
+	Owner       graphql.ID      `json:"ownerId"`
+	Action      *graphql.ID     `json:"actionId,omitempty"`
+	Filter      *graphql.ID     `json:"filterId,omitempty"`
+	// This is being explictly left out to reduce the complexity of the implementation
+	// action *CustomActionsWebhookActionCreateInput
+}
+
+type CustomActionsTriggerDefinitionUpdateInput struct {
+	Id          graphql.ID      `json:"id"`
+	Name        *graphql.String `json:"name,omitempty"`
+	Description *graphql.String `json:"description,omitempty"`
+	Owner       *graphql.ID     `json:"ownerId,omitempty"`
+	Action      *graphql.ID     `json:"actionId,omitempty"`
+	Filter      *graphql.ID     `json:"filterId,omitempty"`
+}
+
+func (client *Client) CreateWebhookAction(input CustomActionsWebhookActionCreateInput) (*CustomActionsExternalAction, error) {
```

Comment:
I remember we talked about it but I'm forgetting why there isn't a GetWebhookAction function?